yb-voyager/src/srcdb/postgres.go

```diff
@@ -940,6 +940,7 @@ var PG_QUERY_TO_CHECK_IF_TABLE_HAS_PK = `SELECT nspname AS schema_name, relname
 FROM pg_class c
 LEFT JOIN pg_namespace n ON n.oid = c.relnamespace
 LEFT JOIN pg_constraint con ON con.conrelid = c.oid AND con.contype = 'p'
+WHERE c.relkind = 'r'  -- Only consider ordinary tables
```

Comment:
why do we need to do this, we should report partitioned tables not having pk as well right?

Reply:
updated to have `p` and `r` both. Previously it was also considering objects like sequences since they also don't have PK on them.